base/pkg/entry.jl

```diff
@@ -519,6 +519,12 @@ function build!(pkgs::Vector, errs::Dict, seen::Set=Set())
     errfile = tempname()
     close(open(errfile, "w")) # create empty file
     code = """
```

Comment:
Windows has a hard limit on the length of this string of 32 kB. `create_expr_cache` passing these values over STDIN to be certain to avoid this limit